glue_job.tf

```diff
@@ -35,22 +37,43 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "glue_resources" {
   }
 }
 
-data "template_file" "signatures_script" {
-  template = file("${path.module}/templates/signatures_job.py.tpl")
-  vars = {
+locals {
+  # Unsupported column types read from CSV files: all of these will be read as 'string'
+  unsupported_input_column_types = [
+    "boolean",
+    "character varying.*",
+    "decimal.*",
+    "hstore",
+    "jsonb",
+    "numeric.*",
+    "timestamp"
+  ]
+
+  # Unsupported columnt types for Redshift: these will be replaced by the mapped type
+  unsupported_output_column_types = {
+    "hstore" = "string"
+    "jsonb" = "string"
+    "numeric\\(3,2\\)" = "decimal(3,2)"
+    "timestamp without time zone" = "timestamp"
+  }
+
+  signatures_script = templatefile("${path.module}/templates/signatures_job.py.tftpl", {
```

Comment:
What's the distinction between the `.tpl` and `.tftpl` suffixes?

Reply:
It's probably just a convention, the more substantial change regarding the template is replacing the use of `template_file` with `templatefile` as explained here: https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file.
Beyond updating to the new standard way of rendering content from a template file, is that `templatefile` supports passing variables that are not strings, like `unsupported_input_column_types` and `unsupported_output_column_types` in this case.